<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameChild</class>
 <widget class="QMdiSubWindow" name="GameChild">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#GameChild{

background-image: url(:/res/Images/Backgrounds/Hanamura.png);
}</string>
  </property>
  <widget class="QLabel" name="grid1_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>5</y>
     <width>780</width>
     <height>350</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>780</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>780</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/Images/Backgrounds/Grid.png);
background-color: rgba(255,255,255, .44);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="grid1_label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>365</y>
     <width>780</width>
     <height>350</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>780</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>780</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/Images/Backgrounds/Grid.png);
background-color: rgba(255,255,255, .44);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Picon5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Picon4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Picon3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Picon2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Picon1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Eicon1">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>140</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Eicon2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>240</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Eicon3">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>340</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Eicon4">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>440</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Eicon5">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>540</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pCap">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>610</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="eCap">
   <property name="geometry">
    <rect>
     <x>895</x>
     <y>10</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="p_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>605</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #37759c;
border: 5px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="e_back">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>5</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #a82828;
border: 5px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Picon1_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, .62);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Picon2_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, .62);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Picon3_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, .62);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Picon4_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, .62);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Picon5_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, .62);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Eicon1_back">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>140</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, .62);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Eicon2_back">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>240</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, .62);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Eicon3_back">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>340</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, .62);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Eicon4_back">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>440</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, .62);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Eicon5_back">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>540</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255, .62);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Eicon5_back</zorder>
  <zorder>Eicon4_back</zorder>
  <zorder>Eicon3_back</zorder>
  <zorder>Eicon2_back</zorder>
  <zorder>Eicon1_back</zorder>
  <zorder>Picon5_back</zorder>
  <zorder>Picon4_back</zorder>
  <zorder>Picon3_back</zorder>
  <zorder>Picon2_back</zorder>
  <zorder>Picon1_back</zorder>
  <zorder>e_back</zorder>
  <zorder>p_back</zorder>
  <zorder>grid1_label</zorder>
  <zorder>grid1_label_2</zorder>
  <zorder>Picon5</zorder>
  <zorder>Picon4</zorder>
  <zorder>Picon3</zorder>
  <zorder>Picon2</zorder>
  <zorder>Picon1</zorder>
  <zorder>Eicon1</zorder>
  <zorder>Eicon2</zorder>
  <zorder>Eicon3</zorder>
  <zorder>Eicon4</zorder>
  <zorder>Eicon5</zorder>
  <zorder>pCap</zorder>
  <zorder>eCap</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMdiSubWindow</class>
   <extends>QWidget</extends>
   <header>qmdisubwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
